matchengine/engine.py

```diff
@@ -645,6 +686,9 @@ def find_trial_matches(self):
             else:
                 logging.info('Matching trial %s' % trial['nct_id'])
 
+            # skip trials that found in trial_match for query_data
```

Comment:
？？？

Reply:
Sorry, the comment location is wrong. This means that MatchEngine will skip trials that found in matched history before matching.